extensions/interactions/PencilCodeEditor/directives/pencil-code-reset-confirmation.component.html

```diff
@@ -14,7 +14,7 @@ <h4 class="modal-title">
           [innerHTML]="'I18N_INTERACTIONS_PENCILCODEEDITOR_CANCEL' | translate">
   </button>
   <button type="button"
-          (click)="confirm()"
+          (click)="confirm(undefined)"
```

Comment:
here and elsewhere, if confirm() expects an arg, let's pass null instead of undefined